pkg/api/volume.go

```diff
@@ -117,7 +117,8 @@ func (v *VolumePortal) CreateVolume() {
 		Context:           ctx.ToJson(),
 	}
 	if _, err = v.CtrClient.CreateVolume(context.Background(), opt); err != nil {
-		log.Error("create volume failed in controller service:", err)
+		CreateVolumeError(ctx, &volume)
+		log.Error("create volume failed in API-server service:", err)
```

Comment:
I am wondering if it's better to move this log message before `CreateVolumeError()` method.